mmv1/products/databasemigrationservice/privateconnection.yaml

```diff
@@ -0,0 +1,123 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'PrivateConnection'
+base_url: 'projects/{{project}}/locations/{{location}}/privateConnections'
+create_url: 'projects/{{project}}/locations/{{location}}/privateConnections?privateConnectionId={{private_connection_id}}'
+self_link: 'projects/{{project}}/locations/{{location}}/privateConnections/{{private_connection_id}}'
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/database-migration/docs/oracle-to-postgresql/create-private-connectivity-configuration'
+  api: 'https://cloud.google.com/database-migration/docs/reference/rest/v1/projects.locations.privateConnections'
+async: !ruby/object:Api::OpAsync
```

Comment:
small cleanup: I think the async block could be trimmed down to:

```
timeouts: !ruby/object:Api::Timeouts
  insert_minutes: 60
  delete_minutes: 60
async: !ruby/object:Api::OpAsync
  operation: !ruby/object:Api::OpAsync::Operation
    base_url: '{{op_id}}'
```

see https://github.com/hashicorp/terraform-provider-google/issues/14976 for more context